enterprise/coderd/dormancy/dormantusersjob.go

```diff
@@ -25,71 +25,49 @@ const (
 
 // CheckInactiveUsers function updates status of inactive users from active to dormant
 // using default parameters.
-func CheckInactiveUsers(ctx context.Context, logger slog.Logger, db database.Store, auditor audit.Auditor) func() {
-	return CheckInactiveUsersWithOptions(ctx, logger, db, auditor, jobInterval, accountDormancyPeriod)
+func CheckInactiveUsers(ctx context.Context, logger slog.Logger, clk quartz.Clock, db database.Store, auditor audit.Auditor) func() {
```

Comment:
I don't even understand the point of this function if the other is also exported

Reply:
Seems like it's just to hardcode the dormancy period